client/src/pages/Login/Login.component.tsx

```diff
@@ -1,23 +1,30 @@
-import { Spacer } from '@chakra-ui/react'
+import { Box, Spacer, Text } from '@chakra-ui/react'
 import { Navigate } from 'react-router-dom'
 import AuthForm from '../../components/AuthForm/AuthForm.component'
 import { useAuth } from '../../contexts/AuthContext'
-import './Login.styles.scss'
 
-const Login = () => {
+const Login = (): JSX.Element => {
   const { user } = useAuth()
 
   if (user) {
     return <Navigate replace to={`/agency/${user.agency.shortname}`} />
   } else {
     return (
-      <div className="login-page">
+      <Box
+        mt="48px"
+        px={{ base: '36px', xs: '48px' }}
+        mx="auto"
+        maxW="calc(320px + 48px * 2)"
+        className="login-page"
```

Comment:
```suggestion

```
Unnecessary line^

Reply:
Let's keep that for now - we might want to further refactor the theming to use a separate file